extension/observer/dockerobserver/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker Observer Extension
+
+**Status: beta**
+
+The `docker_observer` uses a Docker client and queries the Docker Engine API for running containers.
+
+Requires Docker API Version 1.22+.
+
+The collector will need permissions to access the Docker Engine API.
+
+
+### Configuration
+
+#### `endpoint`
+
+The URL of the docker server.
+
+default: `unix:///var/run/docker.sock`
+
+#### `timeout`
+
+The maximum amount of time to wait for docker API responses.
+
+default: `5s`
+
+#### `excluded_images`
+
+A list of filters whose matching images are to be excluded. Supports literals, globs, and regex.
+
+default: `[]`
+
+#### `use_hostname_if_present`
+
+If true, the "Config.Hostname" field (if present) of the docker
```

Comment:
Could be helpful to mention where `Config.Hostname` and `NetworkSettings.IPAddress` are being obtained from (https://docs.docker.com/engine/api/v1.41/#operation/ContainerInspect or similar?)

Reply:
Agreed - I'll add this in.